src/Compilers/Core/Portable/Compilation/OperationWalker.cs

```diff
@@ -193,7 +193,7 @@ public override void VisitEndStatement(IEndStatement operation)
         public override void VisitInvocationExpression(IInvocationExpression operation)
         {
             Visit(operation.Instance);
-            VisitArray(operation.ArgumentsInSourceOrder);
+            VisitArray(operation.ArgumentsInParameterOrder);
```

Comment:
Oh right, this is not related to this PR. Just a bug I wanted to fix a while ago but forgot. Without this change, user's callback for `OperationKind.Argument` will never be invoked for parameter using default value.